js/sliderModel.js

```diff
@@ -70,84 +69,73 @@ define([
         return;
       }
 
-      var items = this.get('_items');
-      var userAnswer = this.get('_userAnswer');
-      for (var i = 0, l = items.length; i < l; i++) {
-        var item = items[i];
-        if (item.value === userAnswer) {
-          item.selected = true;
-          this.set('_selectedItem', item);
-          break;
-        }
+      const items = this.get('_items');
+      const userAnswer = this.get('_userAnswer');
+      const selectedItem = items.find(({ value }) => value === userAnswer);
+      if (selectedItem) {
+        selectedItem.selected = true;
+        this.set('_selectedItem', selectedItem);
       }
 
       this.setQuestionAsSubmitted();
       this.markQuestion();
       this.setScore();
       this.setupFeedback();
-    },
+    }
 
     //This preserves the state of the users answers for returning or showing the users answer
-    storeUserAnswer: function() {
+    storeUserAnswer() {
       this.set('_userAnswer', this.get('_selectedItem').value);
-    },
+    }
 
-    resetUserAnswer: function() {
+    resetUserAnswer() {
       this.set({
         _isAtLeastOneCorrectSelection: false,
         _userAnswer: undefined
       });
       this.selectDefaultItem();
-    },
-
-    deselectAllItems: function() {
-      _.each(this.get('_items'), function(item) {
-        item.selected = false;
-      }, this);
-    },
-
-    isCorrect: function() {
-      var numberOfCorrectAnswers = 0;
-
-      _.each(this.get('_items'), function(item, index) {
-        if (item.selected && item.correct) {
-          this.set('_isAtLeastOneCorrectSelection', true);
-          numberOfCorrectAnswers++;
-        }
-      }, this);
+    }
 
+    deselectAllItems() {
+      this.get('_items').forEach(item => item.selected = false);
```

Comment:
Direct assignment returns aren't allowed
```suggestion
      this.get('_items').forEach(item => (item.selected = false));
```